docs/deploy/set-up-app-registrations.md

```diff
@@ -41,15 +41,18 @@ On your AuthServer page:
       ![Diagram](../images/add_exposed_api_scope_server_app.png)
 
 ## Client App Registration
-**Note** - This client app registration will be used to manually generate the AAD Token required to call AAD protected Web Api as there is no client application in the sample.
-Follow instructions on how to register your client application with Azure Active Directory [here](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app)
+
+> Note:
+>
+> 1. This client app registration will be used to manually generate the AAD Token required to call AAD protected Web API as there is no client application in the sample. Follow instructions on how to register your client application with Azure Active Directory [here](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app)
```

Comment:
This client app registration will be used to manually generate the AAD Token / use BasicAuthClient   to call AAD protected Web API as there is no client application

(whatever the name of Test client should be , just  check with @ikumarapeli-msft)

Reply:
No client supported for now